src/Adapter/OrderMessage/CommandHandler/DeleteOrderMessageHandler.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Adapter\OrderMessage\CommandHandler;
+
+use PrestaShop\PrestaShop\Adapter\OrderMessage\AbstractOrderMessageHandler;
+use PrestaShop\PrestaShop\Core\Domain\OrderMessage\Command\DeleteOrderMessageCommand;
+use PrestaShop\PrestaShop\Core\Domain\OrderMessage\CommandHandler\DeleteOrderMessageHandlerInterface;
+use PrestaShop\PrestaShop\Core\Domain\OrderMessage\Exception\OrderMessageException;
+
+/**
+ * Handles deleting order message using object model
+ *
+ * @internal
+ */
+final class DeleteOrderMessageHandler extends AbstractOrderMessageHandler implements DeleteOrderMessageHandlerInterface
+{
+    public function handle(DeleteOrderMessageCommand $command): void
```

Comment:
missing docblock

Reply:
It's on purpose tbh. :stuck_out_tongue:

Reply:
why? 😄

Reply:
I guess this is because the signature says it all
However @sarjon AFAIK not all IDEs can use the method signature as a replacement for phpDoc :/ so please keep the docblock

This is bothering but makes the project a little bit more inclusive hopefully

Reply:
Which IDEs don't? It feels like supporting IE6, just because there is one guy out there still using it. :/

Reply:
Yes, we need them 👍